examples/run/run.cpp

```diff
@@ -607,16 +681,34 @@ class LlamaData {
         }
 
         nlohmann::json manifest = nlohmann::json::parse(manifest_str);
-        std::string    layer;
+        std::string    sha_model;
+        std::string    sha_template;
```

Comment:
IMO this is a bad approach because ollama uses Go template, not Jinja.

If you want to add this, be aware that most templates won't be detected.

Reply:
Didn't know about this. Considering this I agree that this approach doesn't seem good. 
What do you think? @ericcurtin

Reply:
I'm not an expert on this, but whatever works to make "vnd.ollama.image.template" compatible with llama-run

Reply:
You know the magic model to test this stuff granite-code :)

Reply:
No shame in reading ollama code either to figure out how it gets passed to llama.cpp